app/views/issues/_reports.html.erb

```diff
@@ -0,0 +1,16 @@
+<% reports.each do |report| %>
+	<% details =  report.details.split("--||--") %>
```

Comment:
An extra space. And tabs. And this delimiter should be made a constant or something.